README.md

```diff
@@ -28,6 +28,12 @@ information, vlans, and interfaces, etc.
 * `GetTransceivers()` **show interface transceiver details** (fiber transceivers)
 * `GetGeneric()`: runs any arbitrary command and produces JSON output
 
+Additionally, the library allows "batch" execution of configuration commands.
+
```

Comment:
@tao-sj , extra space.

Reply:
sorry, where is it ?

Reply:
@tao-sj , needs `commands, e.g.` (note the period vs. comma)

![image](https://user-images.githubusercontent.com/3826416/73562014-32ce8f80-4428-11ea-86f9-52ea9368e214.png)

Reply:
oh, updated.